source/reference/minio-mc/mc-cp.rst

```diff
@@ -449,6 +449,38 @@ command only applies to S3-to-S3 copy.
    :start-after: start-versioning-admonition
    :end-before: end-versioning-admonition
 
+Add a ``content-type`` Value
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use :mc-cmd:`mc cp --attr` to add a ``content-type`` value.
```

Comment:
Unclear if this should also have `This command only applies to S3-to-S3 copy` as some of the other examples do. Are there restrictions where you can do this?

Reply:
:thinking: probably. I doubt we persist metadata to disk when copying from S3 to POSIX.